### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by Shmotties** [rastapasta/pokemon-go-xposed#3]
Can a tutorial be released on how to use? Please.

**Comment by exodius48** [comment]
Agreed. I already installed this, no error logs on xposed, but till now i can't reproduce how to know this module works / not.

**Comment by rastapasta** [comment]
Install and activate the module, restart your phone, use your favorite MITM project again (like [pokemon-go-mitm-node](https://github.com/rastapasta/pokemon-go-mitm-node),  [pogo-optimizer](https://github.com/justinleewells/pogo-optimizer) or [mitmdump_decoder](https://github.com/bettse/mitmdump_decoder))!

To see if this module works: the connection to the intercepting proxy will work - even without any custom CA certificate installed.

This is a pretty technical module without any own "functionality" next to simulating an always secure network connection to the API.

Cheers!

**Comment by exodius48** [comment]
@rastapasta I'm interested in this https://github.com/rastapasta/pokemon-go-mitm-node , because on your screenies you can grab more items in poke stop. But i don't understand how to compile it. I'm not a programmeer. Could you share pre-compiled version?  hehe

**Comment by Shmotties** [comment]
Im also not a programmer but i would like for someone to help me privatly maybe 1on1 or maybe make a video showing how you did everything and also how to use pokemon go mitm node?

**Comment by GabrielTK** [comment]
Well, That's a library. If you're not a programmer, I suggest using [this](https://github.com/maierfelix/POGOserver/) for now. I'm currently developing something based on mitm-node
